beatifulArranement.py

```diff
@@ -0,0 +1,30 @@
+#Time Complexity : O(p) where p is the number of permutations which are valid
```

Comment:
You don't know the value of p. Time complexity should be exponential

Reply:
Alright, will take care of that and make the changes.